web/app/configs/capacities.json

```diff
@@ -239,6 +245,13 @@
             "wind": 2938
         }
     },
+    "RS": {
+        "capacity": {
+            "coal": 5644,
+            "hydro": 3006,
+            "unknown": 428
```

Comment:
I will change "unknown" to "gas" then. Just so I understand you correctly @brunolajoie - you suggested to change the value of unknown/gas to 490, or 424/505 ?